packages/clarity-js/types/layout.d.ts

```diff
@@ -10,6 +11,13 @@ export const enum Source {
     CharacterData
 }
 
+export const enum Interaction {
+    Rendered = 0,
+    Visible = 1,
+    Clicked = 2,
+    Input = 3
```

Comment:
can you define these with gaps in between the numbers?

Reply:
done, thanks